app/queues/certificate/certificate-flow.server.ts

```diff
@@ -67,6 +72,8 @@ export const addCertRequest = async ({ rootDomain, username }: AddCertRequest) =
         type: 'exponential',
         delay: 60_000, // start with 1 minute, double each time
       },
+      removeOnComplete: { age: Number(JOB_REMOVAL_FREQUENCY_MS) || JOB_REMOVAL_INTERVAL_MS },
+      removeOnFail: { age: Number(JOB_REMOVAL_FREQUENCY_MS) || JOB_REMOVAL_INTERVAL_MS },
```

Comment:
`age` seems not ms as I read the documentation as below. 
https://docs.bullmq.io/guide/queues/auto-removal-of-jobs
```js
await myQueue.add(
  'test',
  { foo: 'bar' },
  {
    removeOnComplete: {
      age: 3600, // keep up to 1 hour
      count: 1000, // keep up to 1000 jobs
    },
    removeOnFail: {
      age: 24 * 3600, // keep up to 24 hours
    },
  },
);
```

And I think without argument name, `JOB_REMOVAL_INTERVAL_MS` will be the max number of jobs to keep.

Reply:
> `age` seems not ms as I read the documentation as below. https://docs.bullmq.io/guide/queues/auto-removal-of-jobs
> 
> ```js
> await myQueue.add(
>   'test',
>   { foo: 'bar' },
>   {
>     removeOnComplete: {
>       age: 3600, // keep up to 1 hour
>       count: 1000, // keep up to 1000 jobs
>     },
>     removeOnFail: {
>       age: 24 * 3600, // keep up to 24 hours
>     },
>   },
> );
> ```

Yes, age needs to be in seconds; thanks for pointing that out. However, could you please clarify what you mean by:
>And I think without argument name, JOB_REMOVAL_INTERVAL_MS will be the max number of jobs to keep.

Reply:
This is from the doc 

### Keep a certain number of jobs

It is also possible to specify a maximum number of jobs to keep. A good practice is to keep a handful of completed jobs and a much larger value of failed jobs:

```
await myQueue.add(
  'test',
  { foo: 'bar' },
  { removeOnComplete: 1000, removeOnFail: 5000 },
);
```

Reply:
Sorry. Nevermind.I read the code wrong.